spec/controllers/queued_posts_controller_spec.rb

```diff
@@ -31,6 +31,21 @@
     let!(:user) { log_in(:moderator) }
     let(:qp) { Fabricate(:queued_post) }
 
+    context 'not found' do
+      it 'returns json error' do
+        id = qp.id
+        qp.destroy
```

Comment:
When we don't have to check the return value of `destroy` or `save`, we should use `destroy!` or `save!` so that an error is raised instead of having the action fail silently.